Makefile

```diff
@@ -41,17 +41,15 @@ helm_dep_up:
 update_dependencies: setup_helm_repos helm_dep_up update_internals_versions update_docs_versions
 
 .PHONY: get_release_notes
-get_release_notes: create_version_file
+get_release_notes:
+	# Obtain current and previous chart versions
+	$(eval VERSION_NEW=$(shell git branch --remote --sort version:refname | grep -Eo "release-[0-9]+\.[0-9]+\.[0-9]+$$" | tail -n 1 | cut -d - -f 2))
+	$(eval VERSION_OLD=$(shell git branch --remote --sort version:refname | grep -Eo "release-[0-9]+\.[0-9]+\.[0-9]+$$" | tail -n 2 | head -n 1 | cut -d - -f 2))
+	if [ -z "${VERSION_NEW}" ] || [ -z "${VERSION_OLD}" ]; then echo "Can't find the current or previous chart version."; exit 1; fi
```

Comment:
I'm adding "...from the release branches" (see my reply above). Although right now, this is **very unlikely** to fail, as something would have to go very wrong to prevent git from finding the remote `release-x.x.x` branches.